src/server/serviceProvider/transformers/BibloAdmin/getContentPage.transform.js

```diff
@@ -21,6 +21,37 @@ const getContentPageTransform = {
     return this.callServiceClient('bibloadmin', 'getContentPage', {slug});
   },
 
+  getArticleLocations(contentResponse) {
+    const ContentPageFactBox = contentResponse.field_secondary_content || [];
+    const ContentPageLeft = contentResponse.field_content || [];
+
+    if (contentResponse.title) {
+      ContentPageLeft.unshift({
+        widgetName: 'ContentPageTextWidget',
+        widgetConfig: {
+          content: `<h2>${contentResponse.title}</h2>`
+        }
+      });
+    }
+
+    return {ContentPageLeft, ContentPageFactBox};
```

Comment:
Det ligner at ContentPageLeft kan erstattes af this.getSectioLocation(contentResponse) - koden ligner ihvertfald hianden mistænkeligt